### PR TITLE
Replace all `#FFFFFF` with `#000000`

### PR DESCRIPTION
**Pull request by LitoMore**
All `#FFFFFF` brand colors can be changed to `#000000`. White color is not commonly used.

### Note

- [Oh Dear](https://simpleicons.org/?q=ohdear) - The icon needs to be updated, but we can still use black on the old one
- [Zulip](https://simpleicons.org/?q=zulip) - The color `#50ADFF` was picked from its favicon

### CODE REVIEW

File: _data/simple-icons.json
Comment (service-paradis) [inline _data/simple-icons.json:7042]: Looks like we should remove this one for V10. It redirects to https://github.blog/2022-08-15-the-next-step-for-lgtm-com-github-code-scanning/

File: _data/simple-icons.json
Comment (service-paradis) [inline _data/simple-icons.json:8787]: Guidelines mentions `#FF3900`

File: _data/simple-icons.json
Comment (service-paradis) [inline _data/simple-icons.json:12519]: Should plan to remove this one maybe?

**Review by service-paradis — CHANGES_REQUESTED** [review]
I have the feeling that black or white are not really the correct colors for many of them...

File: _data/simple-icons.json
Comment (LitoMore) [reply _data/simple-icons.json:8787]:
```suggestion
            "hex": "FF3900",
```

File: _data/simple-icons.json
Comment (LitoMore) [reply _data/simple-icons.json:7042]: Sure. We can remove those icons with another pull request.

File: _data/simple-icons.json
Comment (LitoMore) [reply _data/simple-icons.json:12519]: np. Let's do this in another pull request. On this pull request, I just want to drop all `#FFFFFFF` color.

File: _data/simple-icons.json
Comment (service-paradis) [inline _data/simple-icons.json:1448]:
```suggestion
            "hex": "5056E5",
```
They use this color a lot on their website. Might be a better choice, what do you think?

File: _data/simple-icons.json
Comment (service-paradis) [inline _data/simple-icons.json:4382]: Looking at https://fluxteam.net/, `#000000` does not really represents the brand. I would go for something more purpleish like `#8D73B0` or one of the dark blue value found as background.

File: _data/simple-icons.json
Comment (service-paradis) [inline _data/simple-icons.json:5670]: Vector logos are available at https://www.hihonor.com/global/brand-guideline11/basics/logo. Could also be added as guidelines. According to this document, `#000000` is the correct color to use in this case!

File: _data/simple-icons.json
Comment (service-paradis) [inline _data/simple-icons.json:6957]: No real use of black for their website. If we want to use a darker color instead of white, we might use the color of their header and footer (`#222222`).

File: _data/simple-icons.json
Comment (service-paradis) [inline _data/simple-icons.json:6979]:
```suggestion
            "hex": "14854F",
```
Might be a better choice in this case. What do you think?

File: _data/simple-icons.json
Comment (service-paradis) [inline _data/simple-icons.json:8921]: Not sure black is really representative for this icon. I think we could probably use `#437291` or `#E76f00`.
What do you think?

File: _data/simple-icons.json
Comment (service-paradis) [inline _data/simple-icons.json:10544]: I think a shade of yellow like `#F9CB61` might be a better choice. What do you think?

**Review by service-paradis — CHANGES_REQUESTED** [review]
Added some more comments. The other changes looks fine to me!

File: _data/simple-icons.json
Comment (LitoMore) [reply _data/simple-icons.json:4382]: I prefer to understand that this icon has no theme color. And the `#000000` would be better. The more default scene to use our icons is a white background, if the icon is white there will be visual problems.

Comment (service-paradis) [reply _data/simple-icons.json:4382]: The color is there to represent the brand color. Not necessarily the icon color (even if it often correspond). In this case, the brand seems to use some color scheme, but never uses black.
We can always ask other @simple-icons/maintainers to weight in on this one!

Comment (runxel) [reply _data/simple-icons.json:4382]: I'm in for the purple!

File: _data/simple-icons.json
Comment (LitoMore) [reply _data/simple-icons.json:6980]: Sure.

File: _data/simple-icons.json
Comment (LitoMore) [reply _data/simple-icons.json:8922]: I think black is better. It's head and hands are in black.

Comment (adamrusted) [reply _data/simple-icons.json:8922]: The blue and orange are both prominently used [on their GitHub profile](https://github.com/openjdk) alongside the black/white/red mascot. Personally, I'd go with `#437291` given it's used more in the GitHub profile picture - and is the bolded text in the text logo.

Comment (LitoMore) [reply _data/simple-icons.json:8922]: I don't think we can tamper with the skin color of their mascot, which is black.

Comment (adamrusted) [reply _data/simple-icons.json:8922]: Much like @service-paradis though, I see the hex we provide as a representation of the brand, more than a representation of the mascot that accompanies it. Given the blue and orange are used prominently (albeit behind the mascot in some) I would say either of those is the 'brand colour' over black. The mascot will _display_ as white/black in our previewer anyway!

Comment (runxel) [reply _data/simple-icons.json:8922]: This is the pale blue for sure given how ["JDK"](https://openjdk.org/groups/build/doc/building.html) is written in it, too!

File: _data/simple-icons.json
Comment (adamrusted) [inline _data/simple-icons.json:8922]:
```suggestion
            "hex": "437291",
```

Just this, and then I think we're good to go on this one!

**Review by adamrusted — APPROVED** [review]
LGTM! Thanks @LitoMore.
Will wait for another of the requested reviewers to confirm they're happy before we merge though.

**Review by runxel — APPROVED** [review]
LGTM! Looks all good to me

**Comment by runxel** [comment]
Only thing is, that LGTM.com is no longer active...
We should remove this in the next version!

**Review by adamrusted — APPROVED** [review]
Let's get this merged in then! I'll get a PR open (if there isn't one already) to remove LGTM @runxel

**Pull request merged**
